apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html

```diff
@@ -88,10 +92,10 @@
                     >
                 </div>
             </mat-grid-tile>
-            <mat-grid-tile colspan="4" rowspan="15" class="responsive-grid-title-left">
+            <mat-grid-tile colspan="{{ sideBarBoxWidth }}" rowspan="{{ sideBarBoxLength }}">
```

Comment:
Consider using `[attr.colspan]` & `[attr. rowspan]` through out this file where either colspan or rowspan is used